tests/unit/lms/validation/_canvas_test.py

```diff
@@ -11,25 +11,117 @@
 )
 
 
-class CommonResponseSchemaTests:
+def response():
+    """Return a mock ``requests`` HTTP response object."""
+    return mock.create_autospec(requests.Response, instance=True, spec_set=True)
+
+
+def canvas_list_files_response_schema():
+    """Return a CanvasListFilesResponseSchema."""
+    response_ = response()
+    response_.json.return_value = [
+        {
+            "content-type": "application/pdf",
+            "created_at": "2018-11-22T08:46:38Z",
+            "display_name": "TEST FILE 1",
+            "filename": "TEST_FILE_1.pdf",
+            "folder_id": 81,
+            "hidden": False,
+            "hidden_for_user": False,
+            "id": 188,
+            "lock_at": None,
+            "locked": False,
+            "locked_for_user": False,
+            "media_entry_id": None,
+            "mime_class": "pdf",
+            "modified_at": "2018-11-22T08:46:38Z",
+            "size": 2435546,
+            "thumbnail_url": None,
+            "unlock_at": None,
+            "updated_at": "2019-05-08T15:22:31Z",
+            "upload_status": "success",
+            "url": "TEST_URL_1",
+            "uuid": "TEST_UUID_1",
+            "workflow_state": "processing",
+        },
+        {
+            "content-type": "application/pdf",
+            "created_at": "2018-10-25T15:04:08Z",
+            "display_name": "TEST FILE 2",
+            "filename": "TEST_FILE_2.pdf",
+            "folder_id": 17,
+            "hidden": False,
+            "hidden_for_user": False,
+            "id": 181,
+            "lock_at": None,
+            "locked": False,
+            "locked_for_user": False,
+            "media_entry_id": None,
+            "mime_class": "pdf",
+            "modified_at": "2018-10-25T15:04:08Z",
+            "size": 1407214,
+            "thumbnail_url": None,
+            "unlock_at": None,
+            "updated_at": "2019-02-14T00:33:01Z",
+            "upload_status": "success",
+            "url": "TEST_URL_2",
+            "uuid": "TEST_UUID_2",
+            "workflow_state": "processing",
+        },
+        {
+            "content-type": "application/pdf",
+            "created_at": "2017-09-08T11:05:03Z",
+            "display_name": "TEST FILE 3",
+            "filename": "TEST_FILE_3.pdf",
+            "folder_id": 17,
+            "hidden": False,
+            "hidden_for_user": False,
+            "id": 97,
+            "lock_at": None,
+            "locked": False,
+            "locked_for_user": False,
+            "media_entry_id": None,
+            "mime_class": "pdf",
+            "modified_at": "2017-09-08T11:05:03Z",
+            "size": 265615,
+            "thumbnail_url": None,
+            "unlock_at": None,
+            "updated_at": "2018-10-19T17:16:50Z",
+            "upload_status": "success",
+            "url": "TEST_URL_3",
+            "uuid": "TEST_UUID_3",
+            "workflow_state": "processing",
+        },
+    ]
```

Comment:
To avoid getting too yak-shavey I'm gonna leave the pre-existing fixtures as they are for now, in this PR I just wanted to separate the test classes so I can more easily add to them